MonoDevelop.MSBuild/Schemas/AspireHostingOrchestration.buildschema.json

```diff
@@ -0,0 +1,21 @@
+{
+  "license": "Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE file in the project root for full license information.",
+  "properties": {
+    "DcpDir": {
+      "description": "The path to the directory containing the DCP tools.",
+      "type": "folder-with-slash"
+    },
+    "DcpExtensionsDir": {
+      "description": "The path to the directory containing the DCP tool extensions.",
+      "type": "folder-with-slash"
+    },
+    "DcpBinDir": {
+      "description": "The path to the directory containing the DCP tool extension binaries.",
+      "type": "folder-with-slash"
+    },
+    "DcpCliPath": {
+      "description": "The path to the DCP CLI executable.",
+      "type": "file"
+    }
```

Comment:
I don't think we want users setting these properties at all. These are more "internal" properties to Aspire.

Reply:
They'll be in completion anyways as they'll be discovered in the imported targets - so I figured they may as well have docs. The MSBuild Editor only hides symbols that use the underscore prefix convention to indicate private/internal use.

Reply:
ok, that makes sense. Thanks for the explanation.

Reply:
Would it make sense to add something like `(for testing/validation scenarios)` to indicate that they aren't properties we'd normally expect to be customized?